app/lib/annual_report/top_statuses.rb

```diff
@@ -2,20 +2,44 @@
 
 class AnnualReport::TopStatuses < AnnualReport::Source
   def generate
-    top_reblogs = base_scope.order(reblogs_count: :desc).first&.id
-    top_favourites = base_scope.where.not(id: top_reblogs).order(favourites_count: :desc).first&.id
-    top_replies = base_scope.where.not(id: [top_reblogs, top_favourites]).order(replies_count: :desc).first&.id
-
     {
       top_statuses: {
-        by_reblogs: top_reblogs&.to_s,
-        by_favourites: top_favourites&.to_s,
-        by_replies: top_replies&.to_s,
+        by_reblogs: formatted_status(most_reblogged_status),
+        by_favourites: formatted_status(most_favourited_status),
+        by_replies: formatted_status(most_replied_status),
```

Comment:
`formatted_status` is a pretty confusing name tbh, it kind of reminds me of the existing `status_content_format` helper but is solely about status IDs…

Reply:
Renamed to `status_identifier` which is maybe slightly more clear. Open to other suggestions if that still doesn't work.